tests/services/test_service_groups.py

```diff
@@ -88,3 +89,100 @@ def test_groups_read(app, groups, group_service, user_pub, anon_identity):
         # Anon does not have permission to search
         with pytest.raises(PermissionDeniedError):
             group_service.read(anon_identity, g.name).to_dict()
+
+
+def test_groups_crud(app, group_service, user_pub):
+    """Test creating, updating and deleting a group by name."""
+
+    payload = {
+        "name": "test-role",
+        "description": "Initial description",
+    }
+
+    item = group_service.create(system_identity, payload).to_dict()
+    assert item["id"] == payload["name"]
```

Comment:
won't repeat the nit, but same pattern for the rest and in other files.

```suggestion
    assert payload["name"] == item["id"]
```